acf-json/group_65f00eaa4af50.json

```diff
@@ -0,0 +1,408 @@
+            },
+            "message": "Only use an emergency banner for its intended purpose.",
+            "new_lines": "wpautop",
+            "esc_html": 0
+        },
+        {
+            "key": "field_65f00eaa0e0b8",
+            "label": "Banner type",
+            "name": "banner_type",
+            "aria-label": "",
+            "type": "select",
+            "instructions": "Select \"No banner\" to turn the banner off.",
+            "required": 1,
+            "conditional_logic": 0,
+            "wrapper": {
+                "width": "",
+                "class": "",
+                "id": ""
+            },
+            "choices": {
+                "No banner": "No banner",
+                "Local emergency": "Local emergency",
+                "National emergency": "National emergency",
+                "National mourning": "National mourning"
+            },
+            "default_value": "No banner",
+            "return_format": "value",
+            "multiple": 0,
+            "allow_null": 0,
+            "ui": 0,
+            "ajax": 0,
+            "placeholder": ""
+        },
+        {
+            "key": "field_65f00edc0e0b9",
+            "label": "Name",
+            "name": "banner_mourning_name",
+            "aria-label": "",
+            "type": "text",
+            "instructions": "",
+            "required": 1,
+            "conditional_logic": [
+                [
+                    {
+                        "field": "field_65f00eaa0e0b8",
+                        "operator": "==",
+                        "value": "National mourning"
+                    }
+                ]
+            ],
+            "wrapper": {
+                "width": "",
+                "class": "",
+                "id": ""
+            },
+            "default_value": "",
+            "maxlength": "",
+            "placeholder": "",
+            "prepend": "",
+            "append": ""
+        },
+        {
+            "key": "field_65f0128a0e0c0",
+            "label": "National emergency title",
+            "name": "banner_national_emergency_title",
+            "aria-label": "",
+            "type": "text",
+            "instructions": "This can just read \"National emergency\", or you can write in a title, e.g. \"Storm Malcolm\"",
+            "required": 1,
+            "conditional_logic": [
+                [
+                    {
+                        "field": "field_65f00eaa0e0b8",
+                        "operator": "==",
+                        "value": "National emergency"
+                    }
+                ]
+            ],
+            "wrapper": {
+                "width": "",
+                "class": "",
+                "id": ""
+            },
+            "default_value": "National emergency",
+            "maxlength": 50,
+            "placeholder": "",
+            "prepend": "",
+            "append": ""
+        },
+        {
+            "key": "field_65f013020e0c1",
```

Comment:
Do we need seperate title fields or could we just have one used by all?

Reply:
I've made them separate fields because I wanted the hint text to be different for local and national.  I couldn't see a way of doing that otherwise.  Summary is shared by local and national.